charts/app-config/values-production.yaml

```diff
@@ -3,6 +3,7 @@
 govukEnvironment: production
 externalDomainSuffix: gov.uk
 publishingServiceDomainSuffix: publishing.service.gov.uk
+assetsDomain: assets-eks.publishing.service.gov.uk
```

Comment:
For testing, this looks good. For the traffic split, we'd presumably need to change this from `assets-eks` to `assets`, right?

Reply:
yup - we'll need to switch that over once we're done testing